account-integrations/safe/src/SafeBlsPlugin.sol

```diff
@@ -50,7 +52,7 @@ contract SafeBlsPlugin is BaseAccount {
         address to,
         uint256 value,
         bytes calldata data
-    ) external payable {
+    ) external payable fromThisOrEntryPoint {
         address payable safeAddress = payable(msg.sender);
```

Comment:
Is `msg.sender` always going to be the safe?

Reply:
Hmm I'm not sure. I think yes *if* the function is always executed via safe's module system. The `msg.sender` here is not part of the changes in this PR, but it looks wrong to me too.

I think the original source of using `msg.sender` here is [safe's 4337 example](https://github.com/safe-global/safe-contracts/blob/810fad9/contracts/test/Test4337ModuleAndHandler.sol#L53). I opened a PR over there to change it to use `this`, curious to see what they say: https://github.com/safe-global/safe-contracts/pull/682.